### PR TITLE
[JOSS Review] Revision of Paper

### PR DESCRIPTION
**Pull request by sappelhoff**
In this PR I revise our paper to address all points that @TomDonoghue mentions in https://github.com/openjournals/joss-reviews/issues/1896#issuecomment-560957025

I tried to be minimal with my changes, while still accommodation the feedback (btw: thanks a lot Tom).

Please feel free to chime in and suggest improvements @jasmainak @agramfort

### CODE REVIEW

File: paper/paper.md [hunk 1]
```diff
@@ -110,12 +112,18 @@ data, it has recently been extended for magnetoencephalography
 [MEG; @Niso2018], electroencephalography [EEG; @Pernet2019], and
 intracranial encephalography [iEEG; @Holdgraf2019].
 In this context, many software packages for neurophysiology data analysis are
-now becoming BIDS compatible.
-For this reason, we developed a dedicated software, ``MNE-BIDS`` with the goal
-to provide a programmable interface for BIDS datasets in neurophysiology.
+now starting to support data organized using the BIDS standard, thereby
+becoming "BIDS compatible".
+Within the Python ecosystem, ``MNE-Python`` [@Agramfort2013] is a major
+software package for neurophysiology data analysis, and extending its
```
Comment (jasmainak) [inline paper/paper.md]: we normally say electrophysiology, it's more precise I guess :-)

File: paper/paper.md [hunk 2]
```diff
@@ -110,12 +112,18 @@ data, it has recently been extended for magnetoencephalography
...
+functionality to support BIDS would be a great benefit for its growing user
+base.
+For this reason, we developed a dedicated Python software ``MNE-BIDS``, with
```
Comment (jasmainak) [inline paper/paper.md]:
```suggestion
For this reason, we developed a dedicated Python software package ``MNE-BIDS``, with
```

File: paper/paper.md [hunk 3]
```diff
@@ -110,12 +112,18 @@ data, it has recently been extended for magnetoencephalography
...
+the goal to provide a programmable interface for BIDS datasets in
+neurophysiology, neatly interfacing with ``MNE-Python``.
```
Comment (jasmainak) [inline paper/paper.md]:
```suggestion
neurophysiology with ``MNE-Python``.
```

**Comment by sappelhoff** [comment]
Don't merge yet - I will try to integrate feedback from @jdkent in https://github.com/mne-tools/mne-bids/issues/318 here as well :+1:

**Comment by jasmainak** [comment]
Also, update the number of contributors to 16 :)

**Comment by sappelhoff** [comment]
@jasmainak based on our conversation on whatsapp, this can be merged now :-)

**Comment by jasmainak** [comment]
Thanks @sappelhoff !

**Pull request merged**